src/pip/_internal/wheel.py

```diff
@@ -64,7 +64,7 @@ def rehash(path, blocksize=1 << 20):
     return (digest, length)
 
 
-def open_for_csv(name, mode):
+def open_for_csv(name, mode, encoding='utf8'):
```

Comment:
`encoding` is not used... You really need to add a test for this.

Reply:
Thanks! I had copied over the changes from my local copy of pip.
Would you have any guidance as to where/how to add a test for this case? The error occurs when installing wheels which have unicode module names.

Reply:
And `encoding` is not a valid `open` keyword argument in Python 2.

Reply:
Thanks for the guidance! I will work on this and get back to you.

Reply:
Hello,
I just made some changes for open and added a test. Will see how the pull request works now (I am not working from a Windows machine at the moment).
Could you help with the wheel test please? I could not figure out how to point a new wheel test to the new unicode_file example in the data folder.
Also, mypy seems to be failing but I am not sure why.

Thanks!

Reply:
Hello,
Any help please?
Many thanks for your support!